pkg/networkmanager/interfaces.js

```diff
@@ -2225,6 +2225,7 @@ PageNetworkInterface.prototype = {
         var self = this;
 
         if (!self.dev) {
+            console.log("Trying to switch off without a device?");
```

Comment:
Yes, this is intentional.  The way the UI is setup, it shouldn't be possible to hit "Off" without `self.dev`, since the button is already "Off".  However, the tests seem to manage to do that, I think.